### ISSUE TITLE
Filter from R/All not working

### ISSUE THREAD

**Issue by jtblakely007** [christianselig/apollo-bugs#57]
Subreddits that have been marked as “Filter from r/All still show up in feed. I do tend to add MANY subreddits to this filter so it’s possible I have exceeded an allowable amount. When I check these subreddits they give me the option to remove the filter, indicating to me that think they are filtered. And yet they show up.

iOS Version 12.3
iPhone XS

Phone not jailbroken 

Reproducible: Yes. My feed is full of subreddits I have tried to exclude. 

Expected behavior: If I exclude a subreddit from r/All, I should not see them in my All feed.

**Comment by jmpinney** [comment]
Same issue here, have a ton of filtered subreddits but they still show up 
Apollo version 1.4.20 
iOS 12.2
iPhone 8

**Comment by jeffreyscottfrench** [comment]
Similar issue - I have requested Apollo to filter specific subreddits repeatedly, and still show up as if the request was never made (option still reads as filter from r/All instead of unfilter). If it helps, 2 subreddits that are stubborn are Formula1 and LiverpoolFC.

Apollo 1.4.20 (pro enabled)
iOS 12.3.1
iPhone 6s

**Comment by mrh-is** [comment]
Same issue here, I also have lots of filtered subreddits. From the subreddit page, I get the blue success message at the top, but then it isn't in my filter list. ~Trying to add it from the filter list gives me a red error message.~ (I'm not getting the error any more; I think that might have just been a typo.)

Apollo 1.4.23
iOS 12.4 (16G5046d) (developer beta)
iPhone X

**Comment by sarthurdev** [comment]
Same issue for me also, I can't add/remove entries in my filter list however the ones that are already in the list *do* get filtered from /r/all.

Apollo 1.4.26
iOS 13 Public Beta 2 (was an issue on iOS 12.3 too)
iPhone 8

**Comment by subjectd** [comment]
Yes, please fix this. I counted the number of subreddits and it only goes up to 100, so it lookes like it was just programmed this way. Im trying to block a bunch of porn without nessisarly blocking all nsfw content the internet was made for experiencing.

**Comment by JerryHobby** [comment]
We all need this.

**Comment by DanielMcC97** [comment]
I agree with Subjectd. Looks like there’s a limit to the number of filtered subreddits. I also use it to filter out porn but can no longer add to the list. Not promising that it’s been 5 months without a response.

**Comment by sarthurdev** [comment]
Any chance of a fix here @christianselig ?

**Comment by rosvall** [comment]
I have the same issue.
Apollo 1.6.4
iOS 13.2

**Comment by ksj01** [comment]
I’m also having this issue. I thought it was a limit issue, so I went into my settings and removed some of my filters and added new ones. Upon refresh, the new ones are gone. I’m really tired of seeing /r/MedicalGore.

**Comment by Slowdive85** [comment]
Same issue. Specifically ShitPostCrusaders as I'm trying to get rid of all meme pages. Tried all tips listed above including logging in/out and blocking directly form the sub.

IOS 13.3.1
APOLLO 1.7.2

**Comment by trienow** [comment]
Are you sure? Since Apollo implemented the local Subreddit-blocks in 1.7 it's been working fine for me. Do your blocks show up under Settings > "Filters & Blocks"?

**Comment by Slowdive85** [comment]
> Are you sure? Since Apollo implemented the local Subreddit-blocks in 1.7 it's been working fine for me. Do your blocks show up under Settings > "Filters & Blocks"?

It doesn’t. I have around 30 or so that work fine. With the previous mentioned, I can enter it manually or from the sub and see it filtered, but after navigating away from the filter page, it disappears and I can see it in All again. Navigating to the sub and filtering, then refreshing the page shows it as unfiltered as well.
